libraries/SITL/SIM_Plane.cpp

```diff
@@ -106,6 +117,116 @@ Plane::Plane(const char *frame_str) :
     }
 }
 
+void Plane::load_coeffs(const char *model_json)
```

Comment:
This is very similar to SIM_Frame's Frame::load_frame_params

Reply:
Could they be combined? Maybe you could pass in the json_search array?